images/ami_kms_key.tf

```diff
@@ -51,10 +62,12 @@ data "aws_iam_policy_document" "ami_kms_doc" {
   statement {
     sid = "Allow use of the key"
 
-    # Wildcards (other than the global "*") are not allowed when specifying
-    # a principal (e.g. "${aws_iam_role.ec2amicreate_role.arn}*"), so instead
-    # we set the principal to "*" and restrict access via the condition
-    # below (which does allow for a wildcard pattern match on the role ARN)
+    # Wildcards (other than the global "*") are not allowed when
+    # specifying a principal
+    # (e.g. "${aws_iam_role.ec2amicreate_role.arn}*"), so instead we
```

Comment:
nifty.

Reply:
That was @dav3r's doing, not mine.  Credit where credit is due.